src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,17 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        // I'm not sure what better, to call 2 times find by email
+        // methoud, or made an instance of find by email result?
+        if (userService.findByEmail(email) != null) {
+            if (userService.findByEmail(email).getPassword().equals(password)) {
```

Comment:
we should always strive for [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) - which stands for Don't Repeat Yourself, including situations like this, when we can put the result of this method in a variable to not call it 2 times 

Also, we can remove those `if` statements, combine their condition into a single one and return it's result right away :)